composer.json

```diff
@@ -38,7 +38,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "1.2-dev"
+            "dev-master": "1.3-dev"
```

Comment:
I guess *if* we are strict this is a MINOR patch as it's a feature so this is correct. But yeah, we could sneak it in as a bugfix :)